commit 3119a0b7c8b479faec78a8a46644f8b509fc5f86 (refs/changes/12/2212/2)
Author: Trent Mick <trentm@gmail.com>
Date:   2017-07-12T16:11:28-07:00 (2 years, 3 months ago)
    
    TOOLS-1807 node updates for july 2017 node.js security releases
    Reviewed by: Todd Whiteman <todd.whiteman@joyent.com>
    Approved by: Todd Whiteman <todd.whiteman@joyent.com>

diff --git a/Makefile b/Makefile
index fa2956c..b901ca5 100644
--- a/Makefile
+++ b/Makefile
@@ -33,7 +33,7 @@ SMF_MANIFESTS = $(shell ls smf/manifests/*.xml)
 NODEUNIT	:= ./node_modules/.bin/nodeunit
 CLEAN_FILES += ./node_modules
 
-NODE_PREBUILT_VERSION=v4.6.1
+NODE_PREBUILT_VERSION=v4.8.4
 ifeq ($(shell uname -s),SunOS)
 	NODE_PREBUILT_TAG=zone
 	# Allow building on other than image sdc-minimal-multiarch-lts@15.4.1.
